spec/pages/email_authentications/invitations/new.rb

```diff
@@ -0,0 +1,54 @@
+module Email_authentications
```

Comment:
To keep with Ruby and Rails conventions, we should name this module `EmailAuthentications`.